resources/application-connector/charts/connector-service/files/certs-backup.sh

```diff
@@ -0,0 +1,27 @@
+set -e
+set -o pipefail
+
+backup() {
+  namespace="$1"
+  secretName="$2"
+  backupSecretName="$secretName-backup"
+
+  echo "Checking existence of $backupSecretName"
+  secret=$(kubectl -n "$namespace" get secret "$backupSecretName" --ignore-not-found)
+  if [ -n "$secret" ]
+  then
+    echo "$backupSecretName exists, removing..."
+    kubectl delete secret "$backupSecretName" -n="$namespace"
+  fi
+
+  echo "Checking existence of $secretName"
+  secret=$(kubectl -n "$namespace" get secret "$secretName" --ignore-not-found)
+  if [ -n "$secret" ]
+  then
+    echo "$secretName exists, backing up to $backupSecretName"
+    kubectl get secret "$secretName" -n "$namespace" -oyaml | sed "s/name:.*$/name: $backupSecretName/" | kubectl apply -n "$namespace" -f -
```

Comment:
You do not clean metadata yaml before import. This is not recommended way to export and import resources with the same metadata. Please use the method I have provided you in the example.